web3deployment/verifyWhiteListAddresses.js

```diff
@@ -0,0 +1,149 @@
+const Web3 = require("web3");
+const fs = require("fs");
+const RLP = require('rlp');
+
+//inputs
+<<<<<<< Updated upstream
+const nodeUrl = "";
+const whiteListContractReference = "0xCb46b40def144e355064d729D0e72DFEA80383a9";
+const whiteListContractVerify = "0x0decdaf699f66f87ea41e9a3ccd7d7d1656c4fa8";
+=======
+const nodeUrl = 'https://mainnet.infura.io';
+const whiteListContractReference = "0xCb46b40def144e355064d729D0e72DFEA80383a9";
+//const whiteListContractVerify = "0xcc92a4462761134c00b3e16a9bd6ce8bfad387b0";
+const whiteListContractVerify = "0x0B4A76f733D98c25F1CD5E771A5D66C31Db69649";
+
+
+>>>>>>> Stashed changes
```

Comment:
@ilanDoron you forgot to remove all the git merge comments

Reply:
Ho man
u are so right.
(Shouldn't have added this script to my PR)